src/avalon_sdk_direct/jrpc_worker_registry.py

```diff
@@ -30,8 +29,10 @@ class JRPCWorkerRegistryImpl(WorkerRegistry):
     """
 
     def __init__(self, config):
-        self.__uri_client = HttpJrpcClient(config["tcf"]["json_rpc_uri"])
+        self.__uri_client = HttpJrpcClient(config["json_rpc_uri"])
```

Comment:
please use config.get[] functio to avoid key not found error. Please replicate in other places as well